collect_app/src/main/java/org/odk/collect/android/configure/qr/AppConfigurationGenerator.kt

```diff
@@ -25,6 +27,20 @@ class AppConfigurationGenerator(
         }.toString()
     }
 
+    fun getAppConfigurationAsJsonWithGoogleDriveDetails(googleAccount: String?): String {
```

Comment:
There is a test for `getAppConfigurationAsJsonWithServerDetails` which is above but not for this method.